tests/doctest/doctest.pro

```diff
@@ -0,0 +1,52 @@
+include($$mac_compiler)
+DESTDIR = $$top_srcdir/bin/
+
+
+OBJECTS_DIR=$$top_builddir/tmp/doctest/
+include($$mzroll_pri)
+TEMPLATE = app
+TARGET = doctest
+
+

```

Comment:
Can you also add:
```
CONFIG+=cmdline
```
here? This will make our `doctest` binary a proper CLI executable.